index.html

```diff
@@ -0,0 +1,165 @@
+          key symptoms, treatment and ways it can be annihilated from the
+          country.
+        </p>
+        <div class="span-align">
+          <h2>2023.03.24(MON) ~ 28(FRI)</h2>
+          <span
+            ><img src="./images/Location.svg" alt="Location" /> International
+            Conference Hall, Ridge</span
+          >
+        </div>
+      </section>
+
+      <!-- Program Outline -->
+      <section class="main-row">
+        <h2 class="main-header">Main Program</h2>
+
+        <div class="main-program">
+          <div class="main-details">
+            <img src="./images/incoming_data.svg" alt="Causes image" />
+            <h3>Causes</h3>
+            <p>
+              The risk of a cholera epidemic is highest when poverty, war or
+              natural disasters force people to live in crowded conditions
+              without adequate sanitation.
+            </p>
+          </div>
+          <div class="main-details">
+            <img src="./images/shaking.svg" alt="Symptoms image" />
+            <h3>Symptoms</h3>
+            <p>
+              Symptoms cause mild or moderate diarrhea are often hard to tell.
+              Others develop more-serious signs usually within a few days of
+              infection.
+            </p>
+          </div>
+          <div class="main-details">
+            <img src="./images/treatment.svg" alt="Treatments image" />
+            <h3>Treatments</h3>
+            <p>
+              Cholera patients should be evaluated and treated quickly. With
+              proper treatment, even severely ill patients can be saved.
+            </p>
+          </div>
+          <div class="main-details">
+            <img src="./images/globe.svg" alt="Prone Areas image" />
+            <h3>Prone Areas</h3>
+            <p>
+              The risk of a cholera epidemic is highest when poverty, war or
+              natural disasters force people to live in crowded conditions
+              without adequate sanitation.
+            </p>
+          </div>
+        </div>
+
+        <div class="center-content">
+          <button class="join-btn">Join TOA Forum 2023</button>
+          <p class="main-footer">See More Program</p>
+        </div>
+      </section>
+
+      <!-- Speakers -->
+      <section id="about" class="featured-speakers">
+        <h3 class="features">Featured Speakers</h3>
+        <div class="experts"></div>
+      </section>
+
+      <!-- Media Partners -->
+      <section class="media-partners">
+        <h2>Partners</h2>
+        <div class="partners">
+          <img src="./images/adom.png" alt="Adom Tv" width="100px" />
+          <img src="./images/tv3.png" alt="Tv3" width="100px" />
+          <img src="./images/Onua.png" alt="Onua Tv" width="100px" />
+          <img src="./images/utv.png" alt="United Tv" width="100px" />
+        </div>
+      </section>
+
+      <!-- Footer -->
+      <section class="footer">&copy 2023 - TOA Forum</section>
+    </section>
+
```

Comment:
- [ ] Please make sure your footer looks like [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by adding `logo` and copy right details into the footer and styling according to the design guidelines

| Cindy Shin design    | Your design |
| ------------- | ------------- |
| <img width="1008" alt="Screenshot 2023-04-21 at 1 21 02 AM" src="https://user-images.githubusercontent.com/44798044/233479106-db215209-cc66-490a-9167-4412386df971.png">| <img width="1510" alt="Screenshot 2023-04-20 at 11 37 50 PM" src="https://user-images.githubusercontent.com/44798044/233479174-c084ea96-90e4-4d81-ae2b-75520cc61c41.png">|